pr-data.csv

```diff
@@ -572,21 +572,21 @@ https://github.com/apache/dubbo,5349c13a36d277a090e1dc68fbe7c3b46d78fc90,dubbo-c
 https://github.com/apache/dubbo,737f7a7ea67832d7f17517326fb2491d0a086dd7,dubbo-config/dubbo-config-api,org.apache.dubbo.config.AbstractInterfaceConfigTest.checkInterfaceAndMethod3,UD,,,
 https://github.com/apache/dubbo,737f7a7ea67832d7f17517326fb2491d0a086dd7,dubbo-config/dubbo-config-api,org.apache.dubbo.config.AbstractInterfaceConfigTest.checkInterfaceAndMethod4,UD,,,
 https://github.com/apache/dubbo,737f7a7ea67832d7f17517326fb2491d0a086dd7,dubbo-config/dubbo-config-api,org.apache.dubbo.config.AbstractInterfaceConfigTest.checkInterfaceAndMethod5,UD,,,
 https://github.com/apache/dubbo,737f7a7ea67832d7f17517326fb2491d0a086dd7,dubbo-config/dubbo-config-api,org.apache.dubbo.config.AbstractInterfaceConfigTest.checkInterfaceAndMethods1,UD,,,
 https://github.com/apache/dubbo,737f7a7ea67832d7f17517326fb2491d0a086dd7,dubbo-config/dubbo-config-api,org.apache.dubbo.config.AbstractInterfaceConfigTest.checkInterfaceAndMethods2,UD,,,
-https://github.com/apache/dubbo,737f7a7ea67832d7f17517326fb2491d0a086dd7,dubbo-config/dubbo-config-api,org.apache.dubbo.config.AbstractInterfaceConfigTest.checkStubAndMock1,UD,,,
-https://github.com/apache/dubbo,737f7a7ea67832d7f17517326fb2491d0a086dd7,dubbo-config/dubbo-config-api,org.apache.dubbo.config.AbstractInterfaceConfigTest.checkStubAndMock2,UD,,,
-https://github.com/apache/dubbo,737f7a7ea67832d7f17517326fb2491d0a086dd7,dubbo-config/dubbo-config-api,org.apache.dubbo.config.AbstractInterfaceConfigTest.checkStubAndMock3,UD,,,
-https://github.com/apache/dubbo,737f7a7ea67832d7f17517326fb2491d0a086dd7,dubbo-config/dubbo-config-api,org.apache.dubbo.config.AbstractInterfaceConfigTest.checkStubAndMock4,UD,,,
-https://github.com/apache/dubbo,737f7a7ea67832d7f17517326fb2491d0a086dd7,dubbo-config/dubbo-config-api,org.apache.dubbo.config.AbstractInterfaceConfigTest.checkStubAndMock5,UD,,,
-https://github.com/apache/dubbo,737f7a7ea67832d7f17517326fb2491d0a086dd7,dubbo-config/dubbo-config-api,org.apache.dubbo.config.AbstractInterfaceConfigTest.checkStubAndMock6,OD-Brit,,,
-https://github.com/apache/dubbo,737f7a7ea67832d7f17517326fb2491d0a086dd7,dubbo-config/dubbo-config-api,org.apache.dubbo.config.AbstractInterfaceConfigTest.checkStubAndMock7,UD,,,
-https://github.com/apache/dubbo,737f7a7ea67832d7f17517326fb2491d0a086dd7,dubbo-config/dubbo-config-api,org.apache.dubbo.config.AbstractInterfaceConfigTest.checkStubAndMock8,UD,,,
-https://github.com/apache/dubbo,737f7a7ea67832d7f17517326fb2491d0a086dd7,dubbo-config/dubbo-config-api,org.apache.dubbo.config.AbstractInterfaceConfigTest.checkStubAndMock9,UD,,,
+https://github.com/apache/dubbo,737f7a7ea67832d7f17517326fb2491d0a086dd7,dubbo-config/dubbo-config-api,org.apache.dubbo.config.AbstractInterfaceConfigTest.checkStubAndMock1,UD,MovedOrRenamed,,https://github.com/apache/dubbo/commit/2d26e11458790c34abc8ef2240e0246e71877d31
```

Comment:
Opening a new PR because of Merge Conflicts. I manually checked the File history to find the commits where the methods were renamed. But the body of the tests was different than the earlier version and I found that the newer version of the tests is not flaky. So I am not adding a new test in the IDOFT PR Data.